ctapipe/core/tool.py

```diff
@@ -16,6 +16,13 @@
     HAS_YAML = False
     pass  # no support for YAML
 
+try:
+    import toml
```

Comment:
Point in case: I think nowadays we want to use `tomli` which is also the library chosen for inclusion in the standard lib for python 3.11: https://peps.python.org/pep-0680/

Reply:
`toml` is already installed as part of ctapipe (msut be a dependency of some other package), but sure I can add this one  if it's nicer. I guess the interface is the same though, so can always wait until python 3.11